groups/rds/variables.tf

```
@@ -103,6 +96,12 @@ variable "dns_zone_is_private" {
   type        = bool
 }
 
+variable "rds_cloudwatch_logs_exports" {
+  default     = ["postgresql", "upgrade"]
+  description = "List of chosen log exports for database RDS Cloudwatch Logs"
+  type        = list(any)
```

Can we tighten up the variable type here:

```suggestion
  type        = list(string)
```